setup.py

```diff
@@ -21,8 +21,9 @@
 from pathlib import Path
 from setuptools import setup
 from setuptools import Extension
-import glob
 
+# Add our setupext package to the path, and import it.
+sys.path.append(str(Path(__file__).parent))
```

Comment:
This is essential, as we are not guaranteed to be running in the CWD of ``setup.py``, and can not guarantee importability of ``setupext``.

Reply:
That is a valid point :+1: